neuro_san/session/external_agent_session_factory.py

```diff
@@ -100,15 +99,15 @@ def create_session_from_location_dict(self, agent_location: Dict[str, str],
         return session
 
     @staticmethod
-    def get_tool_registry(agent_name: str,
-                          manifest_tool_registries: Dict[str, AgentToolRegistry]) -> AgentToolRegistry:
+    def get_agent_network(agent_name: str,
+                          manifest_tool_registries: Dict[str, AgentNetwork]) -> AgentNetwork:
```

Comment:
Rename this method to reflect nomenclature of what is really going on.

Interestingly enough this method is not called anywhere which leads me to believe that somewhere along the way
we have lost the nice error message functionality provided here.